features/step_definitions/common_steps.rb

```diff
@@ -483,7 +483,12 @@ def enable_pool(target)
 end
 
 When(/^I register using "([^"]*)" key$/) do |arg1|
-  regurl = "http://#{$server_ip}/XMLRPC"
+  # register to the proxy, or if none, to the server
+  regurl = if $proxy.nil?
```

Comment:
In ruby there  is also the ? operator, which you could use also. 
```regurl = if $proxy nil? ?   "http://#{$server_ip}/XMLRPC" : ` "http://#{$proxy_ip}/XMLRPC"``

Reply:
I tried ? :  initialy, but it looked a bit too obfuscated...

Reply:
Ok